source/compatibility.txt

```diff
@@ -0,0 +1,7 @@
+.. _kotlin-sync-compatibility:
+
+=============
+Compatibility
+=============
+
+TODO
```

Comment:
C: Generally, I think we comment out the TODO rather than leaving it as rendered text. Applies to all.

Reply:
doesnt matter at this stage but good callout!